apps/_pomodoro_app/realtime/application/Enums/Exceptions.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Probably it's better to use this style: https://javascript.info/custom-errors

Reply:
Yep) Made a custom error class, now I can not only signal that this is a bad argument, but also provide user with additional message